hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/utils/columnSearch.tsx

```diff
@@ -69,8 +69,16 @@ class ColumnSearch extends React.PureComponent {
     filterIcon: (filtered: boolean) => (
       <Icon type='search' style={{color: filtered ? '#1890ff' : undefined}}/>
     ),
-    onFilter: (value: string, record: any) =>
-      record[dataIndex].toString().toLowerCase().includes(value.toLowerCase()),
+    onFilter: (value: string, record: any) => {
+      if (record[dataIndex] !== undefined || record[dataIndex] !== null) {
+          return typeof (record[dataIndex]) === typeof {}
+            ? Boolean (record[dataIndex].find(item => Object.values(item).toString().toLowerCase().includes(value.toLowerCase())) ? true :false)
```

Comment:
Hey sorry my previous comment was not very clear, I meant can we do it as:
```suggestion
            ? Boolean (record[dataIndex].find(item => Object.values(item).toString().toLowerCase().includes(value.toLowerCase())))
```

since Boolean() will actually evaluate the expression and return true/false as is required